optimism.tokenlist.json

```diff
@@ -1367,9 +1367,25 @@
       "name": "Equalizer",
       "symbol": "EQZ",
       "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/logos/EQZ.png",
+      "logoURI": "https://ethereum-optimism.github.io/logos/EQZ.png"
```

Comment:
You have broken the EQZ token listing here by removing the bridge extension. As such CI will never pass. 

Please fix this by adding back the below code above your token listing. 

> `"extensions": {
        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
      }`